RELEASE_NOTES.md

```diff
@@ -1,15 +1,20 @@
 # Version 0.5.2
 
-* Issue #12435 FLOR - Consolidate Streams
-
-   -flort_kn_stc_imodem_instrument,flort_kn_stc_imodem_instrument_recovered,flort_dj_dcl_instrument,
-    flort_dj_dcl_instrument_recovered,flort_dj_cspp_instrument,flort_dj_sio_instrument, and
-    flort_dj_sio_instrument_recovered streams are consolidated into one stream called "flort_sample"
-
-   -flort_m_glider_instrument and flort_m_glider_recovered streams are consolidated into one stream called "flort_m_sample"
-
-   -flort_kn_auv_instrument and flort_kn_auv_instrument_recovered streams are consolidated into one stream
-    called "flort_kn_sample"
+* Issue #12435 FLOR - Consolidate Streams:
+    -flort_sample now replaces:
+        flort_kn_stc_imodem_instrument
+        flort_kn_stc_imodem_instrument_recovered
+        flort_dj_dcl_instrument
+        flort_dj_dcl_instrument_recovered
+        flort_dj_cspp_instrument
+        flort_dj_sio_instrument
+        flort_dj_sio_instrument_recovered
+    -flort_m_sample now replaces:
+        flort_m_glider_instrument
+        flort_m_glider_recovered
+    -flort_kn_sample now replaces:
+        flort_kn_auv_instrument
+        flort_kn_auv_instrument_recovered
```

Comment:
You took away the formatting I provided and now it's jumbled in a markdown view - please refer to a markdown style guide or use the formatting I provided.